src/docs/guides/28-how-to-enable-login-as-user/index.md

```diff
@@ -0,0 +1,289 @@
+---
+title: How to enable Login as user
+slug: how-to-enable-login-as-user
+updated: 2020-02-14
+category: guides
+ingress:
+  This article guides you how to take into use the Login as user feature
+  that allows an operator to log into their marketplace as one of the
+  marketplace users
+published: true
+---
+
+The _Login as user_ feature allows marketplace operators to log into
+their marketplace as a specific user of the marketplace. This helps
+operators to experience their marketplace as their users do and to find
+out what is wrong when their users are reporting problems. The feature
+also comes in handy when a marketplace user asks for help with managing
+their data and listings.
+
```

Comment:
I'd state already here briefly that taking this into use requires action, unless you're using latest FTW.

I would highlight clearly here also the limitations of what you can not do when using this (modify Stripe account details, send messages, initiate or transition transactions). Now that's mentioned at the end of a very technical paragraph, which non-technical operators might just skip.